src/lib/containers/HeaderCard.tsx

```diff
@@ -29,6 +29,27 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
   secondaryLabelLimit,
   isAlignToLeftNav
 }) => {
+  // store old document title and description so that we can restore when this component is removed
+  const descriptionElement:Element|null = document.querySelector('meta[name="description"]')
+  const [docTitle] = useState<string>(document.title)
+  const [docDescription] = useState<string>(descriptionElement ? descriptionElement.getAttribute('content')! : '')
+  useEffect(() => {
+    return function cleanup() {
+      document.title = docTitle
+      if (descriptionElement) {
+        descriptionElement.setAttribute('content', docDescription)
```

Comment:
for future reference, you can use the new optional chaining syntax when a variable might not be defined, e.g. 
 descriptionElement?.setAttribute('content', docDescription)